Checkout/Checkout/Source/Validation/Validators/CardValidator.swift

```diff
@@ -79,6 +79,11 @@ public class CardValidator: CardValidating {
   /// - Parameters:
   ///   - cardNumber: The card number to validate.
   /// - Returns: The card's scheme if successful, else an error.
+  public func validateCompleteness(cardNumber: String) -> Result<ValidationScheme, ValidationError.CardNumber> {
+    logManager.queue(event: .validateCardNumber)
```

Comment:
If we are able to merge all ( validate ) functions into one would be nice, otherwise should we log here `logManager.queue(event: .validateCardNumberWithCompleteness)` to indicate this was logged from `validateCompleteness` method ?

Reply:
They would be equivalent, and following[follow up task](https://checkout.atlassian.net/browse/PIMOB-1364) this would be the `validateCardNumber` method. Suppose each method is used at a different time